com.gitbutler.app.yml

```diff
@@ -0,0 +1,79 @@
+id: com.gitbutler.app
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: git-butler
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-run/gvfsd
+modules:
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.42.5.tar.xz
+        sha256: b64278c1f20b8cfdbfb5ff573c37d871aba74a1db26d9b39f74e8953fe61e749
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORT=GTK
+      - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_MINIBROWSER=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - "*"
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: git-butler
+    buildsystem: simple
+    build-commands:
+      - ar x git-butler.deb
+      - tar xf data.tar.gz
+      - install -Dm755 -t /app/bin usr/bin/git-butler
+      - install -Dm644 -t /app/share/metainfo com.gitbutler.app.metainfo.xml
+      - install -Dm644 usr/share/applications/git-butler.desktop /app/share/applications/com.gitbutler.app.desktop
+      - |
+          for size in 32x32 128x128 256x256@2; do
+            install -Dm644 usr/share/icons/hicolor/${size}/apps/git-butler.png /app/share/icons/hicolor/${size}/apps/com.gitbutler.app.png
+          done
+      - desktop-file-edit --set-icon=com.gitbutler.app --add-category=RevisionControl --set-key=Comment --set-value='The GitButler version control client, backed by Git, powered by Tauri/Rust/Svelte' /app/share/applications/com.gitbutler.app.desktop
+      - desktop-file-validate /app/share/applications/com.gitbutler.app.desktop
+      - rm -r usr control.tar.gz data.tar.gz debian-binary git-butler.deb
+    sources:
+      - type: file
+        dest-filename: git-butler.deb
+        url: https://releases.gitbutler.com/releases/release/0.10.11-640/linux/x86_64/git-butler_0.10.11_amd64.deb
```

Comment:
how will this work on aarch64?

Why not building from source?

Reply:
> how will this work on aarch64?

No, this will not work. They only export x86 deb.

>Why not building from source?

Blocked by https://github.com/flatpak/flatpak-builder-tools/issues/383. It has pnpm deps.